NadekoBot.Core/Modules/Utility/QuoteSlashCommands.cs

```diff
@@ -92,11 +92,11 @@ public async Task QuotePrint([Summary("keyword","Name of the quote")]string keyw
                 if (CREmbed.TryParse(quote.Text, out var crembed))
                 {
                     rep.Replace(crembed);
-                    await ctx.Interaction.EmbedAsync(crembed.ToEmbed(), $"`#{quote.Id}` 📣 " + crembed.PlainText?.SanitizeAllMentions() ?? "")
+                    await ctx.Interaction.EmbedAsync(crembed.ToEmbed(), $"`#{quote.Id} {quote.Keyword.ToLowerInvariant().SanitizeAllMentions()}` 📣 " + crembed.PlainText?.SanitizeAllMentions() ?? "")
```

Comment:
What's the reason for changing this?

Reply:
It is just for clarity, since slash commands don't directly show the quote name